File: Javascript/Final_Value_of_Variable_After_Performing_Operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
     let ans = 0;
    for (let x of operations) {
        if (x.includes("++"))
            ans++;
        else
            ans--;
    }
    return ans;
};

File: Cpp/Final_Value_of_Variable_After_Performing_Operations.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int ans = 0;
        for (string x : operations) {
            if (x.find("++") != string::npos)
                ans++;
            else
                ans--;
        }
        return ans;
    }
};